### DISCUSSION TITLE
Welcome to Hue Discussions!

### DISCUSSION THREAD

**Discussion by romainr** [cloudera/hue#1472] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Akhilsnaik** [comment]
Hey Everyone, 

Myself Akhil 🧑🏻 , I am Part of Hue Community since 6 months , My contributions are limited to javascript vulnerabilities, Usability Issues and contributing to other discussions.

Happy to be Part of this community 🙇‍♂️ 👋

**Comment by sreenaths** [comment]
Hi,

Sreenath here.
I too joined Hue 6 months back. **ER-Diagram, Live components in docs & Precompiled parsers in GetHue** are some of my notable contributions. Im also helping in upgrading UI tech to Vue.js+TS, and in turn converting the UI elements to reusable web-components.

Cheers,
Sreenath
